python/lsst/ap/verify/dataset.py

```diff
@@ -64,10 +61,9 @@ def __init__(self, datasetId):
         try:
             self._dataRootDir = getPackageDir(datasetPackage)
         except pexExcept.NotFoundError as e:
-            raise_from(
-                RuntimeError('Dataset %s requires the %s package, which has not been set up.'
-                             % (datasetId, datasetPackage)),
-                e)
+            raise RuntimeError('Dataset %s requires the %s package, which has not been set up.'
+                               % (datasetId, datasetPackage)) \
+                from e
```

Comment:
Ok, `raise from` is neat and I was not aware of it. Thanks for making me aware!

Although, why can't you just put the `from e` immediately after the parentheses, instead of on a new line?

Reply:
Because then I'm splitting a constructor argument across multiple lines, yet putting part of the argument and external logic on the same line? It seems less readable to me, but I won't claim to be an expert on the arcane rules of Python indentation.

Reply:
Yes there are -- `flake8` complains like crazy unless I get the continuation indent *exactly* right.